seeds.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require_relative './Models/author'
+require_relative './Models/book'
+require_relative './Models/order'
+require_relative './Models/reader'
+require_relative './Models/library'
+require_relative 'seeds'
+
+module Seeds
+  def self.fill_the_library(library, file_name)
+    authors = ['Dan Brown', 'Joanne Rowling', 'A.C. Doyle']
```

Comment:
nit: you can use escape symbols for `%w`
```suggestion
    authors = %w[Dan\ Brown Joanne\ Rowling A.C.\ Doyle]
```